Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -132,4 +134,9 @@ struct FPTIBatchData: Codable {
             case tokenizationKey = "tokenization_key"
         }
     }
+
+    private static func isVenmoAppInstalled() -> Bool {
+        let venmoURL = URL(string: "com.venmo.touch.v2://")!
```

Comment:
Also - avoid `!` force unwrap for safety. If the URL creation fails, we can just not set this boolean for analytics.

Reply:
I thought you wanted to make this always available, true or false, not an optional.